### PR TITLE
fix(scope): Remove duplicate findings

### PR DESCRIPTION
**Pull request by mo-drata**

### CODE REVIEW

File: .github/workflows/drata_iac_scan.yml [hunk 1]
```diff
@@ -10,10 +10,10 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Drata Action
-        uses: drata/compliance-as-code-action@v0.0.98
+        uses: drata/compliance-as-code-action@v0.0.99
```
Comment (bnicoll-drata) [inline .github/workflows/drata_iac_scan.yml:13]: We're up to 102 at the moment https://github.com/drata/compliance-as-code-action/releases/tag/v0.0.102

File: aws/_modules/iam/main.tf [hunk 2]
```diff
@@ -32,7 +32,7 @@ resource "aws_iam_role" "TerraFailIAM_role" {
       {
         Effect = "Allow"
         Principal = {
-          AWS = "*"
+          AWS = "terrafail@user.com"
```
Comment (bnicoll-drata) [inline aws/_modules/iam/main.tf:35]: Minor, but this is inconsistent with the previously defined fake user in aws/_modules/api_gateway/main.tf

File: aws/_modules/lambda/main.tf [hunk 3]
```diff
@@ -83,6 +88,20 @@ resource "aws_kinesis_stream" "TerraFailLambda_stream" {
...
+
+  policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "SNS:Subscribe",
+            "Resource": "${aws_sns_topic_subscription.sac_sns_topic_subscription.arn}/*",
+            "Principal": {"AWS" : "acorn@acorn.com"}
```
Comment (bnicoll-drata) [inline aws/_modules/lambda/main.tf:100]: Was acorn intended here?

File: .github/workflows/drata_iac_scan.yml (hunk 1 above)
Comment (mo-drata) [reply .github/workflows/drata_iac_scan.yml:13]: @bnicoll-drata is that not getting automatically updated here https://github.com/marketplace/actions/drata-compliance-as-code-action?version=v0.0.99? That's where I looked

Comment (bnicoll-drata) [reply .github/workflows/drata_iac_scan.yml:13]: Good point, I don't know how that page gets updated, but the repo release page is the most up-to-date

**Review by bnicoll-drata — APPROVED** [review]

**Review by ashah-acorn — APPROVED** [review]

**Pull request merged**